integrationtest/backend/tck/src/main/java/org/hibernate/search/integrationtest/backend/tck/search/predicate/ObjectExistsSearchPredicateIT.java

```diff
@@ -227,13 +253,15 @@ private void checkDocumentsCreation() {
 			nested = nestedObject.toReference();
 			nestedString = nestedObject.field( "string", f -> f.asString() ).toReference();
 			nestedNumeric = nestedObject.field( "numeric", f -> f.asInteger() ).toReference();
-			nestedX2 = nestedObject.objectField( "nestedX2" ).toReference();
+			nestedX2 = nestedObject.objectField( "nestedX2", ObjectFieldStorage.NESTED ).toReference();
+			nestedNoChild = root.objectField( "nestedNoChild", ObjectFieldStorage.NESTED ).toReference();
```

Comment:
Shouldn't you add a value for this object in `initData` in order for the test to make sense?